src/launcher/docker_executor.cpp

```diff
@@ -148,6 +148,14 @@ int main(int argc, char** argv)
       << flags.usage("Missing required option --mapped_directory");
   }
 
+  if (flags.network_cni_plugins_dir.isNone()) {
+      flags.network_cni_plugins_dir = "/usr/lib/cni/";
+  }
+
+  if (flags.network_cni_config_dir.isNone()) {
+      flags.network_cni_config_dir = "/etc/mesos/cni/net.d";
+  }
```

Comment:
These defaults look a bit odd, as if the agent's CNI flags actually have default values only when used with the docker executor? It seems like the agent flags should be directly used and therefore the operator has to set these explicitly? Will that cause a problem for you?

Right now this is a bit odd:

1. Don't set the agent's CNI flags, this gets you these default values only for docker containerizer / executor, but no defaults for the mesos containerizer. That looks strange.
2. Set the agent's CNI flags, in which case both the docker containerizer / mesos containerizer would use the values?